src/js/components/DataTable/Cell.js

```diff
@@ -25,8 +23,8 @@ const Cell = ({
   primaryProperty,
   rowProp,
   scope,
-  theme,
 }) => {
+  const theme = useContext(ThemeContext);
```

Comment:
```suggestion
  const theme = useContext(ThemeContext) || defaultProps.theme;
```
I think we decided to be consistent with defaultProps fallback.